frontend/www/js/omegaup/components/problem/CollectionListAuthor.vue

```diff
@@ -31,6 +31,7 @@
           :tags="tagsList"
           :sort-order="sortOrder"
           :column-name="columnName"
+          :path="`/problem/collection/author/`"
```

Comment:
nit:

```suggestion
          :path="'/problem/collection/author/'"
```

si no es necesario interpolar, se prefiere usar `''` en vez de ``` `` ```